docs/medical-api/api-reference/patient/list-patients.mdx

```diff
@@ -1,13 +1,21 @@
 ---
-title: "List Patients at Facility"
-description: "Lists all Patients receiving care at the specified Facility."
+title: "List Patients"
+description: "Lists all Patients receiving care."
 api: "GET /medical/v1/patient"
 ---
 
+This is a paginated endpoint. For more information see [pagination](/medical-api/more-info/pagination).
```

Comment:
For this list patients page and any page using pagination should we also show the pagination params. To make it easier for them to make a paginated query without necessarily needing to go to the pagination page

Reply:
We can, but I think it makes it more complicated if we expose each individual parameter. The idea is that users can just use `nextPage` and get all data calling those in sequence.